virttest/utils_test/qemu/__init__.py

```diff
@@ -186,6 +186,35 @@ def clear_win_driver_verifier(driver, vm, timeout=300):
         session.close()
 
 
+@error_context.context_aware
+def check_win_driver(session, driver, timeout=300):
+    """
+    Check if driver is installed and if state is running for windows guest.
+    :param session: VM session
+    :param driver: The driver which needs to check.
+    :param timeout: Timeout in seconds.
+    :return: A tuple (status, output) where status is the driver check result
+    and output is the detail information.
+    """
+    error_context.context("Check %s driver state." % driver,
+                          logging.info)
+    msinfo_cmd = "start /wait msinfo32 /report c:\msinfo.txt"
+    driver_check_cmd = "type c:\msinfo.txt |findstr /i \"%s.sys\" " \
+                       "|findstr /i \"running\"" % driver
+    end_time = time.time() + timeout
+    while time.time() < end_time:
+        session.cmd(msinfo_cmd, timeout)
+        status, output = session.cmd_status_output(driver_check_cmd)
+        if status:
+            time.sleep(4)
+        else:
+            break
+    # If driver is not installed,the output from cmd is empty.
+    if not output:
+        output = "%s driver is not installed." % driver
```

Comment:
when driver is in disabled status,the output is also empty,but actually the driver is installed.

Reply:
Updated, pls have a review. thanks.